tests/storages_tests/test_storages.py

```diff
@@ -23,6 +23,7 @@
 from optuna.storages import RDBStorage
 from optuna.storages import RedisStorage
 from optuna.storages._base import DEFAULT_STUDY_NAME_PREFIX
+from optuna.testing.storage import STORAGE_MODES
```

Comment:
[question]

This order of imports seems not to be alphabetical. Is this intended?

```
>>> sorted(["STORAGE_MODES", "StorageSupplier", "a"])[::-1]
['a', 'StorageSupplier', 'STORAGE_MODES']
```

Reply:
This order is determined by `isort` automatically, so I think it is fine.